src/main/kotlin/com/terraformation/backend/seedbank/model/Accession.kt

```diff
@@ -91,6 +91,7 @@ data class AccessionModel(
     val projectId: ProjectId? = null,
     val receivedDate: LocalDate? = null,
     val remaining: SeedQuantityModel? = null,
+    val remainingNotes: String? = null,
```

Comment:
Is this ok or would it be better to pass the notes into AccessionStore.update as a separate argument instead? Here the notes are only used as input during update, it won't be returned back in the model at any point.